index.html

```diff
@@ -138,7 +138,7 @@ <h3>Frequenty Asked Questions (FAQs)</h3>
         </div>
       </div>
       <br>
-      <p>&copy; LinuxChix India 2016-2017</p>
+      <p>&copy; LinuxChix India 2018-2019</p>
```

Comment:
Why does this have 2019?
Also we did not create the website in 2018. What guidelines are you following? Please share.

Reply:
What I meant was,

If you mention copyrights as `year1-year2` then `year1` is the year in which the website was made and `year2` is the current year. So, basically it means that you have the copyrights to the website since its creation.

The other way is only mentioning the current year.

So, have you followed this scheme of `current_year-next_year` seeing some guidelines? What does it mean?

Reply:
I second @shivan1b !